keeper/msg_server_security_test.go

```diff
@@ -0,0 +1,73 @@
+package keeper_test
+
+import (
+	"cosmossdk.io/math"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	markertypes "github.com/provenance-io/provenance/x/marker/types"
+
+	"github.com/provlabs/vault/types"
+	"github.com/provlabs/vault/utils"
+)
+
+func (s *TestSuite) TestMsgServer_SmallFirstSwapIn_HugeDonation_SwapOut() {
```

Comment:
What is this highlighting, and is this supposed to be demonstrating a specific example? It's just a bit confusing after the huge donation is made. Is there supposed to be an attacker and also a victim?